src/piglet.rs

```diff
@@ -152,7 +152,7 @@ fn setup_logging(matches: &ArgMatches) {
 
 /// Parse the command line arguments using clap
 fn get_matches() -> ArgMatches {
-    let app = Command::new(env!("CARGO_BIN_NAME")).version(env!("CARGO_PKG_VERSION"));
+    let app = clap::Command::new(env!("CARGO_BIN_NAME")).version(env!("CARGO_PKG_VERSION"));
```

Comment:
Something removed the "use clap::Command" and prefixed this with "clap::".
Was that deliberate?